front/app/components/IdeasMap/index.tsx

```diff
@@ -179,296 +93,427 @@ const StyledIdeaMapCard = styled(IdeaMapCard)<{ isClickable: boolean }>`
 
   &.animation-enter {
     opacity: 0;
-    top: calc(${mapHeightMobile} - 130px);
 
     &.animation-enter-active {
       opacity: 1;
-      top: calc(${mapHeightMobile} - 130px - 24px);
     }
   }
 `;
 
-interface Props {
+// Custom styling for Esri map
+const StyledMapContainer = styled(Box)`
+  calcite-action-bar {
+    display: none;
+  }
+
+  calcite-action {
+    display: none !important;
+  }
+
+  .esri-popup__main-container {
+    max-width: 300px !important;
+  }
+`;
+
+export interface Props {
+  mapConfig: IMapConfig;
   projectId: string;
   phaseId?: string;
-  className?: string;
-  id?: string;
-  ariaLabelledBy?: string;
-  tabIndex?: number;
+  ideasList: IIdeaData[];
 }
 
-const getInnerContainerLeftMargin = (
-  windowWidth: number,
-  containerWidth: number
-) => {
-  const leftMargin =
-    Math.round((windowWidth - containerWidth) / 2) - mapMarginDesktop;
-  return leftMargin > 0 ? leftMargin : null;
-};
-
-const initialWindowWidth = Math.max(
-  document.documentElement.clientWidth || 0,
-  window.innerWidth || 0
-);
-const initialContainerWidth =
-  document?.getElementById('e2e-ideas-container')?.offsetWidth ||
-  (initialWindowWidth < maxPageWidth ? initialWindowWidth - 40 : maxPageWidth);
-const initialInnerContainerLeftMargin = getInnerContainerLeftMargin(
-  initialWindowWidth,
-  initialContainerWidth
-);
+const IdeasMap = memo<Props>(
+  ({ mapConfig, projectId, phaseId, ideasList }: Props) => {
+    const theme = useTheme();
+    const localize = useLocalize();
+    const { formatMessage } = useIntl();
+    const { data: authUser } = useAuthUser();
+    const [searchParams] = useSearchParams();
+    const isMobileOrSmaller = useBreakpoint('phone');
+    const isTabletOrSmaller = useBreakpoint('tablet');
+
+    // Create div elements to use for inserting React components into Esri map popup
+    // Docs: https://developers.arcgis.com/javascript/latest/custom-ui/#introduction
+    const startIdeaButtonNode = useMemo(() => {
+      return document.createElement('div');
+    }, []);
+    const ideasAtLocationNode = useMemo(() => {
+      return document.createElement('div');
+    }, []);
+
+    // Map state variables
+    const [esriMapView, setEsriMapview] = useState<MapView | null>(null);
+    const [hoveredLayerId, setHoveredLayerId] = useState<string | null>(null);
+    const [clickedMapLocation, setClickedMapLocation] =
+      useState<GeoJSON.Point | null>(null);
+    const [selectedIdea, setSelectedIdea] = useState<string | null>(
+      searchParams.get('selected_idea_id') || null
```

Comment:
It seems like we have two very similar parameters:
 - `selected_idea_id`, which as far as I can tell is never set in this context, and only seems to be used in the admin post mananger
 - `idea_map_id`, which is being set when I click an idea, but when I refresh the page and enter with that parameter it doesn't automatically select the idea

In [this](https://github.com/CitizenLabDotCo/citizenlab/pull/7054/commits/dcec88c31e048dbe155815cd53433492fb83cbee) commit, I removed the `selected_idea_id` param and added some logic that treats the `idea_map_id` URL param as single source of state and when loading the page first time with this param selected, it will automatically select the idea and zoom to it